src/libeinfo/einfo.h

```diff
@@ -47,6 +47,7 @@ typedef enum
 } ECOLOR;
 
 /*! @brief Returns the ASCII code for the color */
+__attribute__((__deprecated__("this shit is gone yo")))
```

Comment:
sorry, but we can't accept things like this.  ignoring the puerile nature, it's not helpful or clear to anyone who hits/sees this warning.

Reply:
It was merely meant as a message while I was working and not for the final cleaned up/squashed commits.